runtime/onert/backend/cpu/ops/SplitLayer.cc

```diff
@@ -34,7 +34,7 @@ SplitLayer::SplitLayer() : _input(nullptr), _num_splits(0), _axis(0), _outputs()
   // DO NOTHING
 }
 
-void SplitLayer::splitFloat32()
+template <typename T> void SplitLayer::splitGeneric(void)
 {
   nnfw::cker::SplitParams op_params;
   op_params.axis = _axis;
```

Comment:
outputDimsPtr and outputDims varialbes are unused.
I think both are unnecessary.

Reply:
You are right. I have removed the redundant statements in the updated commit. Thanks again.

Reply:
you're welcome : )

Reply:
IMO, looks good. @glistening Could you check this PR? I remember you implemented cpu backend `SPLIT` operation.

Reply:
@dr-venkman `outputDimsPtr` and `outputDims` are not necessary. At early days, I added those lines outputDims might be different. However, in `tflite`, they are always equal as I understand.

Reply:
I have removed the lines from source, as per original suggestion and your feedback. Thank you.